site/content/installation/ngf-images/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "NGINX Gateway Fabric Images"
 description:
-weight: 200
+weight: 100
```

Comment:
Sorry, I meant the index file under the installation directory, not ngf-images directory.